backend/typescript/types.ts

```diff
@@ -75,11 +75,15 @@ export type SchedulingDTO = {
   size?: string;
   isPickup: boolean;
   pickupLocation?: string;
+  dayPart: string;
   startTime: Date;
   endTime: Date;
   status: Status;
   frequency: string;
+  recurringDonationId?: number;
+  recurringEndDate?: Date;
```

Comment:
```suggestion
  recurringDonationEndDate?: Date;
```
nit: can we change it to something more verbose?